pkg/trace/api/normalizer.go

```diff
@@ -4,8 +4,10 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/trace/info"
```

Comment:
This import needs to be moved down, outside of the "standard library" group.

Reply:
Oh weird not sure why my editor put it there. Need to fix the auto-import management. Updated